models/classes/QtiResultsService.php

```diff
@@ -354,4 +355,23 @@ private function getDisplayDate(string $epoch): string
     {
         return tao_helpers_Date::displayeDate($epoch, tao_helpers_Date::FORMAT_ISO8601);
     }
+
+
+    private function prepareItemVariableValue($value, $basetype): string
+    {
+        if ($basetype === 'file') {
+
+            if (extension_loaded('fileinfo')) {
```

Comment:
Not sure if we really need this. [PHP documentation](https://www.php.net/manual/en/fileinfo.installation.php) says:
"
This extension is enabled by default as of PHP 5.3.0. Before this time, fileinfo was a PECL extension but is no longer maintained there. However, versions prior to 5.3+ may use the » discontinued PECL extension.
Windows users must include the bundled php_fileinfo.dll DLL file in php.ini to enable this extension.
"

Reply:
According to OPS channel, we can rely on an existence of that extension by default

Reply:
In any case it's required because the Open Source community might want to run TAO on their own servers/clouds where this extension could not be there. so let's keep it

Reply:
Reverted